src/test/java/org/terasology/flexiblepathfinding/WalkingJPSTest.java

```diff
@@ -0,0 +1,180 @@
+// Copyright 2021 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+package org.terasology.flexiblepathfinding;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.provider.Arguments;
+import org.terasology.flexiblepathfinding.helpers.JPSTestHelper;
+import org.terasology.flexiblepathfinding.helpers.TestPaths;
+import org.terasology.flexiblepathfinding.helpers.TestWorlds;
+import org.terasology.flexiblepathfinding.plugins.basic.WalkingPlugin;
+
+public class WalkingJPSTest {
```

Comment:
Add javadoc similar to to LeapingTests and FallingTests to explain that Walking should find paths with only horizontal steps, and that no vertical steps are allowed?